README.md

```diff
@@ -1,8 +1,8 @@
-# Disposable Cloud Environment (DCE)
+# Disposable Cloud Environment<sup>TM</sup>
```

Comment:
Do <sup> tags render in readthedocs?

Reply:
<img width="1116" alt="Screen Shot 2020-01-31 at 10 52 00 AM" src="https://user-images.githubusercontent.com/22507315/73557922-c21f7700-4417-11ea-9600-e92f65ceb9e2.png">

sure does!